tests/i18n/tests.py

```diff
@@ -1424,8 +1424,17 @@ def test_check_for_language(self):
         self.assertTrue(check_for_language('en'))
         self.assertTrue(check_for_language('en-us'))
         self.assertTrue(check_for_language('en-US'))
+        self.assertTrue(check_for_language('be'))
+        self.assertTrue(check_for_language('be@latin'))
+        self.assertTrue(check_for_language('sr-RS@latin'))
+        self.assertTrue(check_for_language('sr-RS@12345'))
```

Comment:
Looks like there is no standard for `@modifier`, but I'd reject all digits ones.  Or to be more strict, we can only accept iqtelif, euro, cyrillic, abegede, devanagari, latin: https://sourceware.org/git/?p=glibc.git;a=blob;f=localedata/SUPPORTED;h=9bf2a4f3840925c38bdefeaba28d2077835c7f07;hb=HEAD